src/Tester/SerializationTests.cs

```diff
@@ -42,6 +42,31 @@ public void InitializeForTesting()
         }
 
         [TestMethod, TestCategory("BVT"), TestCategory("Functional"), TestCategory("Serialization")]
+        public void SerializationTests_DateTime()
```

Comment:
Would it make the break this test into three with descriptive names?

I appreciate the tests and your effort to include these here, and I know also there are already tests with multiple asserts, so this is a suggestion only.